### ISSUE TITLE
[trtexec] MLPerf bert squad onnx file no longer works with TensorRT 8

### ISSUE THREAD

**Issue by kahmed10** [NVIDIA/TensorRT#1592]
## Description

Previously up until TensorRT 7, trtexec successfully ran MLPerf's bert squad reference onnx model. Now with TensorRT 8, trtexec throws the following error:
 ```
 [11/04/2021-19:29:09] [E] Error[10]: [optimizer.cpp::computeCosts::1855] Error Code 10: Internal Error (Could not find any implementation for node {ForeignNode[440 + (Unnamed Layer* 82) [Shuffle]...node_of_output_start_logits]}.)
[11/04/2021-19:29:09] [E] Error[2]: [builder.cpp::buildSerializedNetwork::417] Error Code 2: Internal Error (Assertion enginePtr != nullptr failed.)
Segmentation fault (core dumped)
 ```
 

## Environment

**TensorRT Version**: 8003
**NVIDIA GPU**:  A100 SXM4 40GB
**NVIDIA Driver Version**: 495.29.05
**CUDA Version**: 11.5
**CUDNN Version**: 
**Operating System**: Ubuntu 18.04.6 LTS
**Python Version (if applicable)**: 
**Tensorflow Version (if applicable)**: 
**PyTorch Version (if applicable)**: 
**Baremetal or Container (if so, version)**: container nvcr.io/nvidia/tensorrt:21.10-py3

## Relevant Files

bert squad onnx file: https://zenodo.org/record/3733910/files/model.onnx

## Steps To Reproduce

```
docker pull nvcr.io/nvidia/tensorrt:21.10-py3
docker run --gpus all -it --rm  nvcr.io/nvidia/tensorrt:21.10-py3
```
Download model file and run trtexec (or download outside and mount volume to docker command above):
```
wget https://zenodo.org/record/3733910/files/model.onnx
trtexec --onnx=model.onnx --verbose 
```
[11/04/2021-20:04:54] [E] Error[10]: [optimizer.cpp::computeCosts::1855] Error Code 10: Internal Error (Could not find any implementation for node {ForeignNode[440 + (Unnamed Layer* 82) [Shuffle]...node_of_output_start_logits]}.)
[11/04/2021-20:04:54] [E] Error[2]: [builder.cpp::buildSerializedNetwork::417] Error Code 2: Internal Error (Assertion enginePtr != nullptr failed.)
Please see attached log for verbose output.
[log.txt](https://github.com/NVIDIA/TensorRT/files/7478343/log.txt)

**Comment by zerollzeng** [comment]
you can try increase workspace size, eg, trtexec --onnx=model.onnx --verbose  --workspace=2048

**Comment by kahmed10** [comment]
@zerollzeng looks like any workspace value >= 64 is yielding correct results now from my initial testing. There may be a better way of explaining to the user that this is the problem, but for now, I can close the issue.
